src/components/Display.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display() {
+  return (
+    <div />
+  );
+}
+
+Display.prototype = {
```

Comment:
- [ ] I think you meant `propTypes` here. `Kindly fix the typo. :pray: As a result of this the `Display` component is not rendered in the parent component. 

![image](https://user-images.githubusercontent.com/64894927/134402410-0aa7a15d-3fb8-4438-b1f7-c09383e2b371.png)